lib/ReactionHandler.js

```diff
@@ -29,16 +29,16 @@ class ReactionHandler extends EventEmitter {
      * Verify a reaction for its validity with provided filters
      * @param {object} msg The message object 
      * @param {object} emoji The emoji object containing its name and its ID 
-     * @param {string} userID The user ID of the user who's reacted 
+     * @param {Member} reactor The member who reacted to this message

```

Comment:
I'd prefer if you made clear it's an Eris Member object:

`* @param {Eris.Member} reactor ...`